### ISSUE TITLE
Support for Node 10

### ISSUE THREAD

**Issue by dachinat** [geerlingguy/ansible-role-nodejs#79]
I have changed

```
- name: Ensure Node.js and npm are installed.
  yum: "name=nodejs-{{ nodejs_version[0] }}.* state=present enablerepo='epel,nodesource'"
```

this to

```
- name: Ensure Node.js and npm are installed.
  yum: "name=nodejs-{{ nodejs_version.split('.')[0] }}.* state=present enablerepo='epel,nodesource'"
```

or it was picking '1' from '10.x'

**Comment by avolkov** [comment]
I can confirm the same issue with Node 10.x 



``TASK [geerlingguy.nodejs : Ensure Node.js and npm are installed.] *********************************************************************************************************************************************************************************
fatal: [fmccv2]: FAILED! => {"changed": false, "msg": "No package matching 'nodejs-1.*' found available, installed or updated", "rc": 126, "results": ["No package matching 'nodejs-1.*' found available, installed or updated"]}
``

**Comment by geerlingguy** [comment]
This was fixed via #80. Thanks for reporting!
